index.html

```diff
@@ -4,18 +4,83 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
-    <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link href="https://fonts.googleapis.com/css?family=Bitter:400,400i&display=swap" rel="stylesheet">
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <!-- Nav bar -->
+    <div class="nav_bar">
+    <div class="logo-image">
+        <img src="img/karma-logo.svg">
+    </div>
+    <ul class="nav_bar_ul">
+        <li ><a class="active"href="#howitworks">How it works</a></li>
+        <li><a href="#store">Store</a></li>
+        <li><a href="#Blog">Blog</a></li>
+        <li><a  href="#Help">Help</a></li>
+        <li><a  href="#Login">Login</a></li>
+    </ul>
+    </div>
+    <!-- Different section -->
+    <section class="introduction">
+        <div class="intro_content">
+            <h1>Introducing Karma</h1>
+            <h3>Bring WiFi with you,every where you go</h3>
+        </div>
+        <div class="button">
+            <button>Learn More</button>
+        </div>
+    </section>
+    <!-- Section two -->
+    <section class="cases">
+        <h1 class="cases_h1">Every one needs a little Karma</h1>
+        <ul>
+            <div class="devices">
+                <li>
+                    <img src="../icon-devices.svg" width="60">
```

Comment:
To make these images load properly you should locate them like this

```
  <img src="img/icon-devices.svg" width="60">
```

I see in the commit history that you spent a while trying to fix this. Don't forget to help to ask for help on Slack when you have trouble! 

It's better than we help with the small stuff like this so you can focus on the bigger problems.